components/SocialMedia.vue

```diff
@@ -0,0 +1,49 @@
+<template>
+  <v-card-actions>
+    <v-container>
+      <v-icon
+        class="mx-5"
+        tag="a"
+        href="https://www.facebook.com/recepcaoimeusp"
+        target="_blank"
+        color="#fff"
+        size="35px"
+        >fa-brands fa-facebook</v-icon
+      >
+
+      <v-icon
+        class="mx-5"
+        tag="a"
+        href="https://www.instagram.com/imeusp.recepcao/"
+        target="_blank"
+        color="#fff"
+        size="35px"
+        >fa-brands fa-instagram</v-icon
+      >
+      <v-icon
+        class="mx-5"
+        tag="a"
+        href="https://discord.gg/qKXAkf8EjE"
+        target="_blank"
+        color="#fff"
+        size="35px"
+        >fa-brands fa-discord</v-icon
+      >
+      <v-icon
+        class="mx-5"
+        tag="a"
+        href="https://www.tiktok.com/@imeusp.recepcao"
+        target="_blank"
+        color="#fff"
+        size="35px"
+        >fa-brands fa-tiktok</v-icon
+      >
```

Comment:
Usando as informações dos dados, podemos fazer um laço para renderizar os ícones.

```suggestion
      <v-icon
        v-for="icon in icons"
        :key="icon.name"
        class="mx-5"
        tag="a"
        :href="icon.link"
        target="_blank"
        color="#fff"
        size="35px"
      >
        {{ icon.name }}
      </v-icon>
```